tests/topotests/lib/common_config.py

```diff
@@ -676,7 +676,7 @@ def generate_support_bundle():
     for rname, rnode in router_list.items():
         logger.info("Generating support bundle for {}".format(rname))
         rnode.run("mkdir -p /var/log/frr")
-        bundle_log = rnode.run("python2 /usr/lib/frr/generate_support_bundle.py")
```

Comment:
If i run that script with your changes with python3 it gives me this:
```
Traceback (most recent call last):
  File "/root/frr-source/tools/generate_support_bundle.py", line 123, in <module>
    processConfFile(lines)
  File "/root/frr-source/tools/generate_support_bundle.py", line 91, in processConfFile
    for line in lines:
  File "/root/frr-source/tools/generate_support_bundle.py", line 22, in <genexpr>
    lines = filter(None, (line.rstrip() for line in supportBundleConfFile))
ValueError: I/O operation on closed file.
```

Reply:
And we need this to pass the CI run, which still uses python2. Thats why i would like the script we call to be fixed, not just the invocation of it.

Reply:
CI needs to be updated to python 3. I don't see supporting python 2 as a requirement anymore given that the language is now EOL. While its use will prevail our support does not have to and should not.